tests/std/tests/P0896R4_ranges_alg_fill_n/test.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <utility>
+
+#include <range_algorithm_support.hpp>
+
+struct instantiator {
+    template <class Out>
+    static constexpr void call() {
+        using ranges::fill_n, ranges::iterator_t;
+        using std::same_as;

```

Comment:
`ranges::iterator_t` and `std::same_as` are unused in this test:
```suggestion
        using ranges::fill_n;
```